src/main/java/PersonalBudget/business/expense/domain/repository/ExpenseRepository.java

```diff
@@ -0,0 +1,9 @@
+package PersonalBudget.business.expense.domain.repository;
+
+import PersonalBudget.business.expense.domain.model.ExpenseEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
I think that if you extend the `JpaRepository` you don't need this annotation because the highest parent of this interface `Repository` is annotated with @\Indexed

Reply:
Exactly, you are right :) 

https://stackoverflow.com/questions/44069367/repository-not-necessary-when-implementing-jparepository